scripts/verify-pkg-main.js

```diff
@@ -36,31 +36,61 @@ if (!isValidCwd) {
 }
 
 let invalidMains = 0;
+let invalidModules = 0;
+let invalidTypes = 0;
 globSync('packages/*/package.json').forEach((jsonPath) => {
   const packageInfo = JSON.parse(fs.readFileSync(jsonPath));
   if (!packageInfo.main) {
     return;
   }
+  verifyPath(packageInfo, jsonPath, 'main');
+  verifyPath(packageInfo, jsonPath, 'module');
+  verifyPath(packageInfo, jsonPath, 'types');
+});
+
+if (invalidMains > 0 || invalidModules > 0 || invalidTypes > 0) {
```

Comment:
```js
function logErrorAndExit(message) {
  console.error(message);
  process.exit(1);
}

if (invalidMains > 0) {
  logErrorAndExit(`${invalidMains} incorrect main property values found; please fix.`);
}
if (invalidModules > 0) {
  logErrorAndExit(`${invalidModules} incorrect module property values found; please fix.`);
}
if (invalidTypes > 0) {
  logErrorAndExit(`${invalidTypes} incorrect types property values found; please fix.`);
}

console.log('Success: All packages with main, module, types properties reference appropriate files!');
```

Reply:
should I also update the `verifyPath` method to use this function as well?

Reply:
It looks like `verifyPath()` is tallying up the number of invalid package properties, so we probably don't want to call `process.exit()` inside of it. Am I understanding that right?

Reply:
Doesn't that mean we shouldn't use it here either? Or simplify the message to just "problems found" in general, since iirc we're already spitting out more specific error messages for each problem encountered?

Reply:
Oh yeah, good point. So maybe `logErrorAndExit()` needs to be updated to something like:

```js
function logError(message) {
  console.error(message);
  process.exitCode = 1;
}
```